reference/data-analysis/index.md

```diff
@@ -1,10 +1,12 @@
 # Data analysis
 
-% TO-DO: Add links to "What is data analysis?"%
-
 This section contains reference information for data analysis features, including:
 
+* [Supplied {{anomaly-detect}} configurations](/reference/data-analysis/machine-learning/supplied-anomaly-detection-configurations.md)
+* [Function reference](/reference/data-analysis/machine-learning/machine-learning-functions.md)
```

Comment:
Do we want to provide more info so folks whether to follow this link? i.e. it's not obvious this is anomaly detection related
```suggestion
* [Function reference for anomaly detection jobs](/reference/data-analysis/machine-learning/machine-learning-functions.md)
```